src/pages/components/inputs.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Inputs
+description: >
+  An input is a form field where the user can enter data.
+package: '@zendeskgarden/react-forms'
+propSheets:
+  - forms/src/elements/FauxInput.tsx
+  - forms/src/elements/common/Field.tsx
+  - forms/src/elements/common/Hint.tsx
+  - forms/src/elements/Input.tsx
+  - forms/src/elements/common/Label.tsx
+  - forms/src/elements/MediaInput.tsx
+  - forms/src/elements/common/Message.tsx
+  - forms/src/elements/Textarea.tsx
```

Comment:
In other places, we've gone with component priority ordering rather than alphabetical. Please align with @rossmoody.